package/kernel/mac80211/patches/ath9k/548-ath9k_enable_gpio_chip.patch

```diff
@@ -126,9 +126,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +	gc->sc = sc;
 +	snprintf(gc->label, sizeof(gc->label), "ath9k-%s",
 +		 wiphy_name(sc->hw->wiphy));
-+#ifdef CONFIG_OF
 +	gc->gchip.parent = sc->dev;
-+#endif
```

Comment:
Will it affect the x86 target?

Reply:
I don’t see why it would.

Reply:
The x86 target didn't enable CONFIG_OF. There are some ath9k usb devices.